### DISCUSSION TITLE
New Products confusion

### DISCUSSION THREAD

**Discussion by glaidler** [codatio/codat-docs#760] [General]
Hi, I see that you've re-organised your products, but there seems to be fewer integrations with Accounting systems.

The "Supported Integrations" for the "Sync for Commerce" product is listed below, which is a much smaller list than the previous list for the accounting api.  Is this correct?

![image](https://github.com/codatio/codat-docs/assets/1322247/e75da05e-91ac-43a8-9b60-7e8ba2fb3607)

**Comment by mcclowes** [comment]
Hi @glaidler,

Our general integration lists were previously just presented grouped by the source of the data, but we didn't always signpost everywhere each integration's compatibility with our products. [This was the old state of play](https://knowledge.codat.io/coverage/products/30acbb85-95f4-40b3-85c3-6f4e3bcc861f) so we have actually recently expanded our support for Sync for Commerce. Because we've built a lot of domain specific logic in this product so you don't have to, it doesn't have the full array of accounting platform we've integrated more simply into.

We still do have a wide array of accounting integrations, as you can see in our [Lending API](https://docs.codat.io/lending/overview#supported-integrations).

**Reply by glaidler** [reply]
Thanks - are you saying that the same account api will still work it's just the grouping that has changed?  

For example, would I still be able to sync with freshbooks?

**Reply by glaidler** [reply]
@mcclowes Hi is my understanding above correct?

**Comment by glaidler** [comment]
@mcclowes Hi is my understanding above correct?

**Comment by dcoplowe** [comment]
Hi @glaidler,

Sorry for the delay in response. Today, Sync for Commerce supports five integrations. The reason Sync for Commerce (SfC) does not support all of Codat's accounting/commerce integrations is because the integration either does not work with SfC or Codat does not yet support a SfC data type for the integration.

It might be useful to speak to someone from our commercial team as they will be able to better support you with these sorts of questions. Let me know and I can start making arrangements.

Many thanks,

David
